crates/cxx-qt-lib-headers/include/std_types.h

```diff
@@ -0,0 +1,23 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+#pragma once
+
+#include <QtGlobal>
+
+// For versions less than Qt 6 we need to manually register the std numerics
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
+#include <QtCore/QMetaType>
+
+static const int i8 = qRegisterMetaType<::std::int8_t>("::std::int8_t");
```

Comment:
Seems that https://doc.qt.io/qt-6/qmetatype.html#registerConverter-1 could also be used to convert to qt numbers?

Reply:
Appears that `qRegisterMetaType` is enough